### PR TITLE
Download User Messages

### PR DESCRIPTION
**Pull request by mdhorn**
Changes proposed in this pull request:
- Download Pre and Post Installer messages
TODO: Display the messages as pop-ups

### CODE REVIEW

File: tui/install.go [hunk 1]
```diff
@@ -25,6 +30,26 @@ type InstallPage struct {
...
+
+// Messenger is an array of language string used to create
+// an end user message for display
+type Messenger struct {
+	Messages []*LangString `yaml:"install-msg,omitempty"`
+}
+
+const (
+	//installDataURLBase = "https://cdn.download.clearlinux.org/releases/28090/clear/config/image/.data"
+	installDataURLBase = "http://mhorn-linux.jf.intel.com/pre-release/.data/"
```
Comment (reaganlo) [inline tui/install.go]: "http://mhorn-linux.jf.intel.com/pre-release/.data/"

File: tui/install.go [hunk 2]
```diff
@@ -138,3 +164,46 @@ func newInstallPage(tui *Tui) (Page, error) {
...
+		return
+	}
+
+	var lines []string
+	found := false
+	for _, message := range result.Messages {
+		line := strings.TrimSpace(message.Default)
+		if len(line) > 0 {
+			// We found at least one line with non-whitespace
+			found = true
```
Comment (reaganlo) [inline tui/install.go]: If we are not skipping empty lines why do we need this flag "found"?

Comment (mdhorn) [reply tui/install.go]: This is fixed

File: tui/install.go (hunk 1 above)
Comment (mdhorn) [reply tui/install.go]: Fixed this to the proper URL, not a test.

**Pull request merged**